RetailCoder.VBE/Common/TimerHook.cs

```diff
@@ -1,14 +1,16 @@
 using System;
 using System.Diagnostics;
+using System.Runtime.InteropServices;
 using Rubberduck.Common.WinAPI;
+using NLog;
 
 namespace Rubberduck.Common
 {
     public class TimerHook : IAttachable, IDisposable
```

Comment:
Is this class even in use? It is referenced in the `RubberduckIocInstaller` but by time RD loads, the class is never instantiated nor its methods called....

Reply:
... indeed, that does appear to be the case!   I love wasting my time ;-)

Reply:
I think it is a remenant of the old timed logger interceptor, which has been removed because there has been the feeling that the inability to proply use reflection on the intercepted inspections (to get their type) was too much of a problem.

Reply:
Ok, if there's no objection, I will revise the PR to delete the file altogether and remove it from the IoC installer.

Reply:
sounds good. If it's never used, it should be deleted 👍